src/com/team2502/robot2018/RobotLocalizationCommand.java

```diff
@@ -0,0 +1,105 @@
+package com.team2502.robot2018;
+
+import com.team2502.robot2018.trajectory.localization.IRotationalLocationEstimator;
+import com.team2502.robot2018.trajectory.localization.ITranslationalLocationEstimator;
+import com.team2502.robot2018.trajectory.localization.ITranslationalVelocityEstimator;
+import edu.wpi.first.wpilibj.command.Command;
+import org.joml.ImmutableVector2f;
+
+
+public class RobotLocalizationCommand extends Command implements ITranslationalLocationEstimator, ITranslationalVelocityEstimator, IRotationalLocationEstimator
```

Comment:
I would go as far as calling this an abuse of the command system WPILib provides us. Consider making a `Notifier`.

Reply:
http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/Notifier.html